README.md

```diff
@@ -11,39 +11,60 @@ The documentation is deployed at [Use Python API](https://omero-guides.readthedo
-
-   * Install [Docker](https://www.docker.com/) if required
-   * Create a virtual environment and install repo2docker from PyPI.
-   * Clone this repository
-   * Run ``repo2docker``. 
-   * Depending on the permissions, you might have to run the command as an admin
-   * Run the commands:
-
-    ```
-    pip install jupyter-repo2docker
-    git clone https://github.com/ome/omero-guide-python.git
-    cd omero-guide-python
-    repo2docker .
-    ```
-
-* Building locally using ``conda`` and ``Jupyter``:
-
-    * Create a conda environment i.e. ``conda env update -n stardist -f binder/environment.yml``
-    * Activate the newly created environment
-
-    The following steps are only required if you want to run the notebooks
-    * If you have Anaconda installed:
-        * Start Jupyter from the Anaconda-navigator
-        * Select the notebook you wish to run and select the ``Kernel>Change kernel>Python [conda env:stardist]``
-    * If Anaconda is not installed:
-        * In the environment, install ``jupyter`` e.g. ``pip install jupyter``
-        * Add the virtualenv as a jupyter kernel i.e. ``ipython kernel install --name "stardist" --user``
-        * Open jupyter notebook i.e. ``jupyter notebook`` and select the ``stardist`` kernel or ``[conda env:stardist]`` according to what is   available
+## Run the notebooks
+
+### Running on cloud resources
+
+* [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ome/omero-guide-python/master?filepath=notebooks)
+* [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ome/omero-guide-python/)
+
+The OMERO server used will need to have [websockets support](https://docs.openmicroscopy.org/omero/latest/sysadmins/websockets.html) enabled.
+
+
+
+### Running in Docker
+
+
+Alternatively, if you have Docker installed, you can use the [repo2docker](https://repo2docker.readthedocs.io/en/latest/)
+tool to run this repository as a local Docker instance:
+
+    $ git clone https://github.com/ome/omero-guide-python
+    $ cd omero-guide-python
+    $ repo2docker .
+
+Then follow the instructions that are printed after the Docker image is built.
+
+### Running locally
+
+
+Finally, if you would like to install the necessary requirements locally,
+we suggest using conda:
+
+Install Anaconda https://www.anaconda.com/products/individual#Downloads
+
+Then, create the environment:
+
+    $ git clone https://github.com/ome/omero-guide-python
+    $ cd omero-guide-python
+    $ conda env create -n omero-guide-python -f binder/environment.yml
+
+and activate the newly created environment:
+
+    $ conda activate omero-guide-python
+
+The following steps are only required if you want to run the notebooks
+
+* If you have Anaconda installed:
+  * Start Jupyter from the Anaconda-navigator
+  * Select the notebook you wish to run and select the ``Kernel>Change kernel>Python [conda env:omero-guide-python]``
+* If Anaconda is not installed:
```

Comment:
``If Anaconda is not installed (e.g. you use miniconda):``

Otherwise, it is a bit confusing, as you clearly instructed above: ``Install Anaconda``

Reply:
I am going to rephrase since miniconda could be used too